.gitignore

```diff
@@ -77,3 +77,9 @@ vagrant/puppet/modules/
 vagrant/puppet/.tmp
 .librarian
 .vagrant
+
+#Dirs created in dev
+OpenOversight/db_repository/
```

Comment:
So the model changes that made are not actually db-level changes, which is why no migration was required (I think). As far as I know, the sqlalchemy orm lets you declare relationships in this way without requiring schema changes. Both the models I touched already had FK declarations, but the addition of a relationship declaration just lets you traverse the relationships as if they are properties of the returned object. Which is to say, I don't think anything about how flask-migrate is being used is incorrect.